feature-extensions/splunk/setup.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python2
+import subprocess
+import argparse
+import os.path
```

Comment:
This is now unused and failing the lint check, please remove.

Reply:
Sure, will configure it